src/stories/PageBanner.stories.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { PageBanner, Text, Avatar } from 'components';
+
+storiesOf('PageBanner', module).add('PageBanner', () => (
+  <div>
+    <div class="sb-page-wrapper">
+      <Text
+        className={'sb-component-group-heading'}
+        typeScale="h1"
+        color="purple"
+        weight="fontWeight-bold"
+      >
+        PageBanner
+      </Text>
+
+      <Text
+        className={'sb-component-group-description'}
+        typeScale="Body"
+        lineHeight="lineHeight-default"
+      >
+        PageBanner component can take any content and display it as a
+        dismissable section. The banner can be configured to not be dismissable.
+        A wrapping class can be provided in order to match the content's width
+        to the wrapper of the page's content. The banner will always fill the
+        available width. Once a banner is dismissed it cannot be brought back.
+        The containing page can determine whether the banner shows at all by
+        conditionally rendering the component, but otherwise the open "state" of
+        the banner is managed by the component itself. The parent page can
+        perform actions when the banner is closed by using the
+        <code>onClose</code> prop.
+      </Text>
```

Comment:
I used the template found on Buttons and Text (looks great). Not sure how much explanation is needed for components, but I went with "a lot" :) Let me know if it's too much.

Reply:
This was perfect. The more the better... By the way you'll see we just deployed a new version:
http://components.bounties.network